Keanu/src/pages/login/login.html

```diff
@@ -30,4 +30,7 @@
   <div padding>
     <button block ion-button (click)="login($event)" >Login</button>
   </div>
+  <div padding>
```

Comment:
Is it possible to make this button less prominent or smaller?
Also can we add a message that says something like "don't have an account? Register here"